Libraries/LibJS/Runtime/Value.cpp

```diff
@@ -57,6 +57,9 @@ String Value::to_string() const
         if (is_nan())
             return "NaN";
 
+        if (is_infinity())
+            return as_double() < 0 ? "-Infinity" : "Infinity";
```

Comment:
I know, if `as_double()` is negative infinity, it's lower than _any_ number, not just 0 - if there's a better way to differentiate between +/- inf in C++, please let me know :)